docs/v3/api-ref/rest-api/server/schema.json

```diff
@@ -19738,6 +19754,26 @@
                 ],
                 "title": "FlowRunResponse"
             },
+            "FlowRunResult": {
+                "properties": {
+                    "input_type": {
+                        "type": "string",
+                        "const": "flow_run",
+                        "title": "Input Type",
+                        "default": "flow_run"
+                    },
+                    "id": {
+                        "type": "string",
+                        "format": "uuid",
+                        "title": "Id"
+                    }
+                },
+                "type": "object",
+                "required": [
+                    "id"
+                ],
+                "title": "FlowRunResult"
+            },
```

Comment:
this appears to be here because https://github.com/PrefectHQ/prefect/pull/18338 did not run pre-push hooks